CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.3.0] - 2016-08-12
```

Comment:
Can you please add a markdown link to the diff between v3.2.0 and v3.3.0 like the other versions? See the bottom of this file for examples.